phoenix-pherf/src/main/java/org/apache/phoenix/pherf/util/PhoenixUtil.java

```diff
@@ -274,4 +274,34 @@ public void updatePhoenixStats(String tableName, Scenario scenario) throws Excep
         logger.info("Updating stats for " + tableName);
         executeStatement("UPDATE STATISTICS " + tableName, scenario);
     }
+    
+    /**
+     * Get explain plan for a query
+     *
+     * @param query
+     * @return
+     * @throws SQLException
+     */
+    public String getExplainPlan(Query query) throws SQLException {
+        Connection conn = null;
+        ResultSet rs = null;
+        PreparedStatement statement = null;
+        StringBuilder buf = new StringBuilder();
+        try {
```

Comment:
You might want to consider using the JdbcSession() to handle all the boiler plate try/catch/finally stuff on the connection. 

Check out testRWWorkload()
https://github.com/apache/phoenix/blob/master/phoenix-pherf/src/it/java/org/apache/phoenix/pherf/DataIngestIT.java